ruby/vampires.rb

```diff
@@ -0,0 +1,91 @@
+# vampires.rb 
+# DBC 4.4 Solo Challenge
+
+# Ask how many employees will be processed in each pass
+puts "How many employees will be processed?"
+emp_count = gets.chomp.to_i
+x = 0
+
+# Loop the code until all employees are processed.
+until x >= emp_count
+
+	puts "What is your name?"
+	name = gets.chomp 
+
+	puts "How old are you?"
+	age = gets.chomp
+
+	puts "What year were your born?"
+	birth_year = gets.chomp
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you?"
+	garlic_bread = gets.chomp
+
+	puts "Would you like to enroll in the company’s health insurance?"
+	insurance = gets.chomp
+
+
+	age_calc = 2017 - birth_year.to_i

```

Comment:
When checking something like a year consider an approach that keeps us from needing to change our code every new year.

Take a look the Ruby `Time` class.  `Time.new.year`